var/spack/repos/builtin/packages/lm-sensors/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class LmSensors(MakefilePackage):
+    """The lm-sensors package provides user-space support for the
+    hardware monitoring drivers in Linux. """
+
+    homepage = "https://github.com/groeck/lm-sensors/"
+    url = "https://github.com/groeck/lm-sensors/archive/V3-4-0.tar.gz"
+
+    version('3-4-0', '1e9f117cbfa11be1955adc96df71eadb')
+    version('3-3-5', '42dcbc32c61133f5dbcdfd7ef8d0ee3c')
```

Comment:
have you tested all? Otherwise, one can just start with one version, e.g. the latest

Reply:
Yes, I've tested all the included versions.